atlassian_jwt_auth/aio/key.py

```diff
@@ -0,0 +1,25 @@
+import asyncio
+
+import aiohttp
+
+from atlassian_jwt_auth.key import PEM_FILE_TYPE, \
+    HTTPSPublicKeyRetriever as _HTTPSPublicKeyRetriever
```

Comment:
I strongly prefer to avoid `\` where possible. So instead this should be:

```
from atlassian_jwt_auth.key import (
    PEM_FILE_TYPE,
    HTTPSPublicKeyRetriever as _HTTPSPublicKeyRetriever,
)
```